src/js/flows/getZealot.ts

```diff
@@ -31,7 +31,7 @@ export const getZealot = (): Thunk<Zealot> => (
   getState,
   {createZealot}
 ) => {
-  const conn = Current.mustGetConnection(getState())
+  const conn = Current.getConnection(getState())
```

Comment:
This should stay mustGetConnection I think? If not, we should check for the presence of conn.

Reply:
Ah yes, thanks. I had changed this while debugging but forgot to switch back